spyder/config/main.py

```diff
@@ -93,6 +93,8 @@
               'cpu_usage/timeout': 2000,
               'use_custom_margin': True,
               'custom_margin': 0,
+              'use_custom_cursor_blinking': False,
+              'custom_cursor_blinking': 1000,

```

Comment:
@goanpeca in the header of this file (`main.py`), it is written:
> Note: Leave this file free of Qt related imports [...]

So I don't know how I can link `CURSORBLINK_OSDEFAULT`, whose value must be obtained with QApplication... Do you have any idea? Maybe another solution could be to set the default value when creating the spinbox within the `configdialog.py` module instead of here? Something like:
```
cursor_spin = self.create_spinbox( 
        "", _("ms"), 'custom_cursor_blinking',
        default = QApplication.cursorFlashTime(),
        min_ = 0, max_ = 5000)
```